.github/workflows/smoke.yml

```diff
@@ -96,11 +96,52 @@ jobs:
         run: |
           mv ./ci_pyproject_headed.toml ./pyproject.toml;
           pipenv run pytest --fx-executable=/Volumes/Firefox/Firefox.app/Contents/MacOS/firefox -n 4 . || TEST_EXIT_CODE=$?
-          mv -r artifacts/* artifacts-mac/ || true
+          mv artifacts/* artifacts-mac/ || true
```

Comment:
You're right, `-r` is fake. Can we change it to `-n`? I don't want to clobber anything.

Reply:
ok! on it